src/FlexTable.jl

```diff
@@ -243,14 +243,37 @@ end
 # Private fields are never exposed since they can conflict with column names
 Base.propertynames(t::FlexTable, private::Bool=false) = columnnames(t)
 
+
+Base.vcat(t::FlexTable) = t
```

Comment:
`Base` seems opinionated that `vcat` doesn't alias its inputs:

```julia
julia> v = [1,2,3]
3-element Array{Int64,1}:
 1
 2
 3

julia> vcat(v) === v
false
```

Reply:
You're right, of course, I should have checked. I've amended the PR to behave more similar to Base.